.github/workflows/ci.yml

```
@@ -1,136 +1,113 @@
 
-      - name: Build & push frontend image
-        uses: docker/build-push-action@v3
-        with:
-          context: ./frontend
-          file: ./frontend/Dockerfile
-          push: true
-          tags: |
-            ghcr.io/${{ github.repository_owner }}/employee-management-frontend:${{ github.sha }}
-            ghcr.io/${{ github.repository_owner }}/employee-management-frontend:latest
+          tags: ${{ vars.DOCKER_USERNAME }}/workforceops-${{ matrix.working-directory }}:${{ github.sha }}
+          tags: ${{ vars.DOCKER_USERNAME }}/workforceops-${{ matrix.working-directory }}:latest
```

actionlint (1.7.12)

[error] key "tags" is duplicated in "with" section. previously defined at line:107,col:11. note that this key is case insensitive

(syntax-check)